Assets/Scripts/SceneLoader.cs

```diff
@@ -0,0 +1,70 @@
+using Sirenix.OdinInspector;
+using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Celeritas
+{
```

Comment:
It should be obvious (I know), but would be nice to have a /// comment describing the responsibilities of this class. 10/10 don't need to, just thought I'd mention